### PR TITLE
Add note to FAQ about cache directory

### PR DESCRIPTION
**Pull request by mschristiansen**
Just had a problem trying to use my own package set and it took me a while before I had it narrowed down and starter looking for the cache dir. Hope this note in the README can help someone else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +124,11 @@ Not right now. We might add this feature in future, but for now, consider either
 
 - Adding your dependency to the package set if possible, or
 - Creating your own custom package set
+
+### Why are my changes not updated in my package set?
+
+Package sets are cached in `.psc-package`. If you are changing a
```
Comment (paf31) [inline README.md]: When you say changing a package set, do you mean manually? Could you please clarify that? Also, could you please move the command into its own block-level element?


Thanks!

Comment (paf31) [reply README.md]: Also, maybe point out that we cache based on Git tag names?

**Comment by mschristiansen** [comment]
I've added a more precise explanation of how the cache works and moved code to into separate block. What do you think about having a flag on the update command to pull a package set without using the cache? E.g. `psc-package update --force` will clear cache and fetch everything anew.

File: README.md [hunk 2]
```diff
@@ -124,3 +124,15 @@ Not right now. We might add this feature in future, but for now, consider either
 
 - Adding your dependency to the package set if possible, or
 - Creating your own custom package set
+
+### Why are my changes not updated in my package set?
+
+Package sets are cached based on a git reference (e.g. tag or branch)
```
Comment (paf31) [inline README.md:130]: Just to be clear, this only happens if you're using a branch, or something like master, right? The typical usage should be with _tags_, where this shouldn't happen. Maybe we should spell that out.

Comment (mschristiansen) [reply README.md:130]: You _can_ delete tags. Not good practice, but it happens and would create the same problem.

Comment (mschristiansen) [reply README.md:130]: Yes, I was using a branch, while figuring out how to create a package set and then tagged it once things worked.

**Comment by paf31** [comment]
Thanks very much!

**Pull request merged**